src/Control/Distributed/Process/Tests/Closure.hs

```diff
@@ -363,19 +378,23 @@ testSpawnSupervised TestTransport{..} rtable = do
     [node1, node2]       <- replicateM 2 $ newLocalNode testTransport rtable
     [superPid, childPid] <- replicateM 2 $ newEmptyMVar
     thirdProcessDone     <- newEmptyMVar
+    linkUp               <- newEmptyMVar
 
     forkProcess node1 $ do
       us <- getSelfPid
       liftIO $ putMVar superPid us
-      (child, _ref) <- spawnSupervised (localNodeId node2) (waitClosure 1000000)
+      (child, _ref) <- spawnSupervised (localNodeId node2) (sendPidClosure us `seqCP` staticClosure expectClosure)
```

Comment:
I'm not sure about the line length in a file that don't have a strong policy about what it should be,
I've split this line into two, but didn't try to make it <80 chars.